src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/PropertyExpression.cs

```diff
@@ -38,55 +38,55 @@ private void OnPropertyChanged()
 
             if (Property.Type.IsEnumEx())
             {
-                CreateDataTypeExpressionIfNotCompatible(() => new StringExpression());
+                DataTypeExpression = new StringExpression();
```

Comment:
I removed CreateDataTypeExpressionIfNotCompatible because it checked only type and not nullability. This caused bug that when you chosen column to Price and next to NullablePrice then filter conditions have not changed. In example app all was OK. I can fix it but need example when this actually crashes.